tests/test_docker_compose_base.py

```
@@ -70,30 +68,30 @@ def create_production_compose_content():
-        result = subprocess.run(
+        """Start Qdrant service using docker-compose."""
+        return subprocess.run(
             ["docker", "compose", "-f", str(compose_file), "up", service_name, "-d"],
+            check=False,
             capture_output=True,
             text=True,
             cwd=temp_dir,
         )
-        return result
 
     def stop_qdrant_service(self, compose_file, temp_dir, remove_volumes=True):
```

Ruff (0.12.2)

Boolean default positional argument in function definition

(FBT002)